src/coral_models/data.py

```diff
@@ -118,7 +124,7 @@ def clean_dataset(
     # transcriptions, as they do not have an influence on the pronunciation of the
     # words.
     non_standard_characters_regex = re.compile(
-        f"[^{re.escape(cfg.characters_to_keep)}]"
+        f"[^{re.escape(cfg.model.characters_to_keep)}]"
```

Comment:
Hmm i only now just noticed that we have moved `characters_to_keep` to be a model-level config. In case we have different characters to keep for each model, won't be harder to compare the results between models. Something like,

```
model_a_prediction = "han ringede til Helmut Köhl"
model_b_prediction = "han ringede til Helmut Kohl"
```

Stupid example, but do you catch my drift?

Reply:
Hm, yeah good point. I think this was due to the Whisper models being able to cope with casing as well. But in terms of comparing the models, I guess we just need to have a uniform preprocessing method during evaluation? Because keeping the casing during training would give a more useful model, and it still allows an apples-to-apples evaluation by, say, removing the bad characters and making everything lower case.

Reply:
Some discussion of this dilemma here: https://github.com/huggingface/community-events/tree/main/whisper-fine-tuning-event#pre-processing